scripts/python-skywater-pdk/skywater_pdk/liberty.py

```diff
@@ -388,8 +432,21 @@ def top_write(lines):
     top_write([''])
     top_write(['}'])
     top_fout.close()
-    print("   Finish writing", top_fpath, flush=True)
-    print("")
+    msg_info("   Finish writing", top_fpath)
+    msg_info()
+
+
+def deps(library_dir, lib, corner, ocorner_type, icorner_type, cells, output_directory):
+    output_directory_prefix = None if output_directory else "timing"
+    output_directory = output_directory if output_directory else library_dir
+    top_out_path     = os.path.join(output_directory, top_liberty_file(lib, corner, ocorner_type, output_directory_prefix))
+    common_data_path = os.path.join(output_directory, output_directory_prefix, "{}__common.lib.json".format(lib))
+    top_data_path    = os.path.join(output_directory, top_corner_file(lib, corner, icorner_type, output_directory_prefix))
+    cell_paths_lst = [
```

Comment:
can we keep this as a generator? `( ... for .. in )` or does this need to be materialized as a list at that point?